lib/SILGen/SILGenPoly.cpp

```diff
@@ -490,6 +490,28 @@ ManagedValue Transform::transform(ManagedValue v,
     }
   }
 
+  // - upcasts for structs
```

Comment:
"for collections" to be more precise

Reply:
Ah absolutely, sorry I was fixed on "generic struct" for some reason while writing that comment...